src/lib/sdk.ts

```diff
@@ -110,10 +116,14 @@ class FlutterSDK {
 
       if (flutterPath) {
         flutterPath = await getRealPath(flutterPath);
+        flutterPath = flutterPath.trim();
         if (flutterPath.endsWith(join('bin', 'flutter'))) {
-          this._sdkHOme = flutterPath.replace(join('bin', 'flutter'), '');
+          this._sdkHome = flutterPath.replace(join('bin', 'flutter'), '');
+        } else {
+          this._sdkHome = flutterPath;
         }
-        log(`flutter command path => ${flutterPath}`);
+        this._flutterCommand = join(this._sdkHome, 'bin', 'flutter');
```

Comment:
`this._flutterCommand = join(this._sdkHome, 'bin', 'flutter');`  can be just `this._flutterCommand = flutterPath` ?

Reply:
This depends on your answer on the previous comment ;)